unit-10/exercise8/main.js

```diff
@@ -0,0 +1,144 @@
+// Exercise 1
+const getMultiplyBoard = (number) => {
```

Comment:
Hàm arrow function nếu chỉ có 1 param thì bỏ `()` đi
```suggestion
const getMultiplyBoard = number => {
```